src/components/Slide.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { useEffect, useRef, useState } from 'react'
+import styled from 'styled-components';
+import RightDirection from '@/assets/icons/RightDirection';
+import LeftDirection from '@/assets/icons/LeftDirection';
+
+const Slide = ({ movies }) => {
+    const TOTAL_SLIDES = 9;
+
+    const [currentSlide, setCurrentSlide] = useState(0);
+    const slideRef = useRef(null);
+
+    // const [validNextBtn, setValidNextBtn] = useState(true);
+    // const [validPrevBtn, setValidPrevBtn] = useState(true);
+
+    const NextSlide = () => {
+        if (currentSlide >= TOTAL_SLIDES) {
+            setCurrentSlide(0);
+        } else {
+            setCurrentSlide(currentSlide + 1);
+        }
+    };    
+    
+    const PrevSlide = () => {
+        if (currentSlide === 0) {
+            setCurrentSlide(TOTAL_SLIDES); 
+        } else {
+            setCurrentSlide(currentSlide - 1);
+        }
+    };
+    
+    useEffect(() => {
+        slideRef.current.style.transition = 'all 0.5s ease-in-out';
+        slideRef.current.style.transform = `translateX(-${currentSlide}00%)`;
+    }, [currentSlide]);
+
+    return (
+        <Container>
+            <Arrows>
+                <Button onClick={PrevSlide}>
+                    <LeftDirection />
+                </Button>
+                <Button onClick={NextSlide}>
+                    <RightDirection />
+                </Button>
+            </Arrows>
+            <SlideWrapper ref={slideRef}>
+                {movies?.map((movie) => (
+                    <PosterImage key={movie.id} src={movie.poster} alt="title" />
+                ))}
```

Comment:
저번 주 강사님께서 코드리뷰 해주신 노션 참고해서 아래와 같이 수정할 수 있겠네요!
```suggestion
                {movies?.map(({id, poster}) => (
                    <PosterImage key={id} src={poster} alt="title" />
                ))}
```